test/test_tree_transformer.py

```diff
@@ -1507,6 +1512,62 @@ def test_ruff_fix_pass_file_name(tmp_path, monkeypatch) -> None:
     assert transformer(dedent(source)) == dedent(expected)
 
 
+def test_ruff_format(tmp_path, monkeypatch) -> None:
+    config_file = tmp_path / "ruff.toml"
+    config_file.write_text(tomli_w.dumps({"format": {"quote-style": "single"}}))
+    monkeypatch.chdir(tmp_path)
+    transformer = TreeTransformer(ruff_format=True)
+
+    source = """
+    mode = "format"
+    """
+
+    expected = """
+    mode = 'format'
+    """
+
+    assert transformer(dedent(source)) == dedent(expected)
```

Comment:
I feel kind of stupid. Why is this failing?

Reply:
The block comments `"""` start with a newline. Ruff removes the leading newline. You'll need to make the code start on the same line as the comment to fix this :)

Reply:
Another one like this. This fails:

```python
source = """
import time, asyncio
mode = "format"
"""
```

But this as well:

```python
source = """import time, asyncio
mode = "format"
"""
```

Can you help me out again?